### ISSUE TITLE
Maven repository with artefacts

### ISSUE THREAD

**Issue by dopoljak** [quarkiverse/quarkus-discord4j#19]
Hi,

I've noticed that there doesn't seem to be a Maven repository with artifacts available. Has a version been released yet, or are there plans for an upcoming release?

Thanks,
Domagoj

**Comment by indiealexh** [comment]
+1 Same question. also happy to contribute.

**Comment by gsmet** [comment]
I think the main problem to solve would be to be able to run the tests. Right now, they require a token which makes them hard to automate.
Or maybe Discord provides a test server on which we can create a token? If so we could probably make it work. If you folks can figure this out, that could help.

I created a PR here to update to Quarkus 3.7: https://github.com/quarkiverse/quarkus-discord4j/pull/20

@cottoncammy any thoughts?

**Comment by dopoljak** [comment]
Can you maybe make build without test first make it configurable and set it in action pipeline until we figure out something?

**Comment by gsmet** [comment]
I was able to get CI working at least for PRs that are directly in the repo (it won't work for PRs coming from forks as we need secrets and secrets are not available in pull requests coming from forks).

I stumbled upon https://github.com/quarkiverse/quarkus-discord4j/issues/21 which doesn't look completely blocking for releasing an alpha.

@cottoncammy I could try to push a `0.0.1` for people to play with it if you think it's reasonably ready?
